CHANGELOG.md

```diff
@@ -5,14 +5,22 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [UNRELEASED]
+## [0.9.0] - 2023-04-23
```

Comment:
This header change is not needed; the CI will auto-generate it.  The "Added" and "Changed" blocks are sufficient.